### ISSUE TITLE
gazelle: Unable to find dependency label for UIKit

### ISSUE THREAD

**Issue by AttilaTheFun** [cgrindel/swift_bazel#181]
Hi! I'm trying to follow your quickstart guide and seeing this when trying to run tidy:
```
ogan@Logans-MBP Stetho % bazel run //:tidy               
INFO: Analyzed target //:tidy (26 packages loaded, 130 targets configured).
INFO: Found 1 target...
Target //:tidy up-to-date:
  bazel-bin/tidy
INFO: Elapsed time: 0.270s, Critical Path: 0.01s
INFO: 4 processes: 4 internal.
INFO: Build completed successfully, 4 total actions
INFO: Running command line: bazel-bin/tidy :swift_update_pkgs :update_build_files
INFO: Analyzed target //:swift_update_pkgs (58 packages loaded, 9314 targets configured).
INFO: Found 1 target...
Target //:swift_update_pkgs up-to-date:
  bazel-bin/swift_update_pkgs-runner.bash
  bazel-bin/swift_update_pkgs
INFO: Elapsed time: 0.519s, Critical Path: 0.02s
INFO: 5 processes: 5 internal.
INFO: Build completed successfully, 5 total actions
INFO: Running command line: bazel-bin/swift_update_pkgs
INFO: Analyzed target //:update_build_files (0 packages loaded, 2 targets configured).
INFO: Found 1 target...
Target //:update_build_files up-to-date:
  bazel-bin/update_build_files-runner.bash
  bazel-bin/update_build_files
INFO: Elapsed time: 0.117s, Critical Path: 0.01s
INFO: 2 processes: 2 internal.
INFO: Build completed successfully, 2 total actions
INFO: Running command line: bazel-bin/update_build_files
gazelle: Unable to find dependency label for Combine
gazelle: Unable to find dependency label for UIKit
```

Is there a way to handle these kinds of system frameworks?

**Comment by cgrindel** [comment]
I have not run into the Comine one yet. I am chasing down when `UIKit` and `AppKit` are magically included. I should have something next week.

**Comment by cgrindel** [comment]
This should be fixed by #205. That PR generates lists of the frameworks that ship with the macOS and iOS SDKs.
